### DISCUSSION TITLE
Password reset

### DISCUSSION THREAD

**Discussion by thomaslochet** [hrfee/jfa-go#122] [Q&A]
Hi, I really cannot understand how the password reset fonction work. How did I manage to reset a password ?
I'm sorry if this question is stupid but I really need help :)

**Comment by hrfee** [comment]
On the Jellyfin login page, press "Forgot Password". A box will come up, enter your username and press reset. A message pops up saying "Ask the admin to read the PIN from <file path>" or similar. jfa-go monitors the directory of that file path. You set this path in jfa-go Settings > Password Resets > Jellyfin directory. If you use Docker, make sure that jfa-go can access the directory. It reads the PIN and either:
* Sends the PIN to the user. They enter it after pressing OK on the reset page on Jellyfin. When entered, their password is set to the PIN. This is the default
* Sends a link to the user. When clicked, their password is set to the PIN, which they are given. (Enable "Use reset link instead of PIN" in jfa-go Settings > Password Resets)
* Sends a link to the user. When clicked, they can enter their new password. (Enable "Use reset link instead of PIN" and "Set password through link")

To use any of these, you need a way to contact the user. This can be via Email (SMTP or Mailgun), or a chatbot on Discord, Telegram or Matrix.
